### PR TITLE
Assignment_Q1_Amarja

### PR DESCRIPTION
**Pull request by AmarjaBTechnocredits**
Calculator program.Please review

### CODE REVIEW

File: JavaAssignments/src/amarja/CalculatorProg.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		public static void main(String[] args) {
+		CalculatorProg a = new CalculatorProg();
+		System.out.println("Number 1= "+ Integer.parseInt(args[0]));
+		System.out.println("Number 2= "+ Integer.parseInt(args[1]));
+		a.addition(Integer.parseInt(args[0]),Integer.parseInt(args[1]));
+		a.subtraction(Integer.parseInt(args[0]),Integer.parseInt(args[1]));
+		a.multiplication(Integer.parseInt(args[0]),Integer.parseInt(args[1]));
+		a.division(Float.parseFloat(args[0]),Float.parseFloat(args[1]));
+	}
+		void addition (int num1,int num2)
```
Comment (mkanani13) [inline JavaAssignments/src/amarja/CalculatorProg.java:13]: it is always better to return result, caller may expect some result while calling it.

**Review by mkanani13 — APPROVED** [review]
it is always better to return result, caller may expect some result while calling it.

I am approving it for now.

File: JavaAssignments/src/amarja/CalculatorProg.java (hunk 1 above)
Comment (AmarjaBTechnocredits) [reply JavaAssignments/src/amarja/CalculatorProg.java:13]: ok will follow always

Comment (mkanani13) [reply JavaAssignments/src/amarja/CalculatorProg.java:13]: Amarja, change it here also. change return type void to int or float.

**Review by mkanani13 — CHANGES_REQUESTED** [review]

**Pull request merged**